<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- Start republish node to decompress JPEG image -->
<!--
  <node name="republish" pkg="image_transport" type="republish"
    respawn="false" output="screen" args="compressed in:=camera/image raw out:=camera/image_raw">
  </node>
-->

  <!-- Publish fixed map->odom tf -->
<!--
  <node name="fixed_map" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 /map /odom 25" />
-->

  <!-- Publish fixed odom->base_link tf -->
<!--
  <node name="fixed_map2" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 /odom /base_link 25" />
-->

  <!-- Publish fixed base_link->camera tf -->
  <node name="fixed_map3" pkg="tf" type="static_transform_publisher" args="0.085 0 0.07 1.5708 0 1.5708 /base_link /camera 25" />

<!--
  <node name="fixed_map4" pkg="tf" type="static_transform_publisher" args="0 0 .0335 0 -1.570796 -3.14159 /map /ar_marker_0 25" />
-->

  <!-- Start ALVAR tracker node -->
  <arg name="marker_size" default="6.7" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="cam_image_topic" default="/camera/image_raw" />
  <arg name="cam_info_topic" default="/camera/camera_info" />
  <arg name="output_frame" default="/base_link" />

  <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)"/>
</launch>
